redis-pq-demo/README.md

```diff
@@ -0,0 +1,29 @@
+# Example run-time
+
+This is a demo that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
```

Comment:
"and uses REDIS as a resilient..." (drop the`to`).
Put a link to REDIS so one can jump to documentation.

Reply:
a link to redis as a whole? or the acapy-redis-events-queue plugin?

Reply:
Both. The more information they can get to easier is better.